src/videojs-http-streaming.js

```diff
@@ -826,6 +828,11 @@ class HlsHandler extends Component {
       delete this.tech_.hls;
     }
 
+    if (this.mediaSourceUrl_ && window.URL.revokeObjectURL) {
```

Comment:
This was a memory leak, we were never revoking the mediaSources url blob. See https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL and https://developer.mozilla.org/en-US/docs/Web/API/URL/revokeObjectURL